From fef13a371fc91f6d9b42518126c2a8b459fdf0e8 Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Thu, 02 May 2013 18:17:38 +0000
Subject: Remove use of G_DISABLE_DEPRECATED

This now trips up on GStaticRecMutex; I'm not going to bother to port,
realistically no one is actively maintaining this code anymore.
---
diff --git a/activation-server/Makefile.am b/activation-server/Makefile.am
index efb18a8..ffa90f6 100644
--- a/activation-server/Makefile.am
+++ b/activation-server/Makefile.am
@@ -17,7 +17,6 @@ INCLUDES=					\
 	$(SERVER_CFLAGS)			\
 	$(WARN_CFLAGS)				\
 	-DSERVER_CONFDIR=\"$(sysconfdir)\"	\
-	-DG_DISABLE_DEPRECATED			\
 	-DG_LOG_DOMAIN=\"Bonobo-Activation-Server\" \
 	$(NULL)
 
diff --git a/configure.in b/configure.in
index 70a4e26..6be2bca 100644
--- a/configure.in
+++ b/configure.in
@@ -132,7 +132,7 @@ AC_SUBST(SERVER_LIBS)
 AC_SUBST(SERVER_CFLAGS)
 
 if test "$enable_maintainer_mode" = "yes"; then
-	DISABLE_DEPRECATED_CFLAGS="-DG_DISABLE_DEPRECATED -DG_DISABLE_SINGLE_INCLUDES"
+	DISABLE_DEPRECATED_CFLAGS="-DG_DISABLE_SINGLE_INCLUDES"
 	AC_SUBST(DISABLE_DEPRECATED_CFLAGS)
 fi
 
--
cgit v0.9.2
